src/main/java/com/ebay/sonar/test/github/TestGithub.java

```diff
@@ -4,6 +4,29 @@
 
 	public void testJava(){
 		System.out.println("test");
+		 if ( 3 != 3 ) { // always false
+  	System.out.println("test");
+}
+if ( 3 == 3 && 3 == 3 ) { // if the first one is true, the second one is too
+ 	System.out.println("test");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AS106)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.vip.qa.ebay.com/coding_rules#rule_key=squid%3AIndentationCheck)